competitions/cloud-cover/benchmark-tutorial.ipynb

```diff
@@ -0,0 +1,2586 @@
+    "We can convert the bounding box to longitude and latitude using `pyproj`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyproj"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def lat_long_bounds(filepath):\n",
+    "    \"\"\"Given the path to a GeoTIFF, returns the image bounds in latitude and\n",
+    "    longitude coordinates.\n",
+    "\n",
+    "    Returns points as a tuple of (left, bottom, right, top)\n",
+    "    \"\"\"\n",
+    "    with rasterio.open(filepath) as im:\n",
+    "        bounds = im.bounds\n",
+    "        meta = im.meta\n",
+    "    # create a converter starting with the current projection\n",
+    "    current_crs = pyproj.CRS(meta[\"crs\"])\n",
+    "    crs_transform = pyproj.Transformer.from_crs(current_crs, current_crs.geodetic_crs)\n",
+    "\n",
+    "    # returns left, bottom, right, top\n",
+    "    return crs_transform.transform_bounds(*bounds)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Image coordinates (lat, long):\n",
+      "Start: (2.948221298028172, 35.44628398518643)\n",
+      "End: (2.9946024439490517, 35.492417498478574)\n"
+     ]
+    }
+   ],
+   "source": [
+    "left, bottom, right, top = lat_long_bounds(example_chip[\"B04_path\"])\n",
+    "print(\n",
+    "    f\"Image coordinates (lat, long):\\nStart: ({left}, {bottom})\"\n",
+    "    f\"\\nEnd: ({right}, {top})\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### True color image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can make a composite image from the three visible bands (blue, green, and red) to visualize a high-quality, true color image. To show the true color image, we'll use the `xarray` and `xrspatial` packages designed for Sentinel-2 satellite data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import xarray\n",
+    "import xrspatial.multispectral as ms"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_xarray(filepath):\n",
+    "    \"\"\"Put images in xarray.DataArray format\"\"\"\n",
+    "    im_arr = np.array(Image.open(filepath))\n",
+    "    return xarray.DataArray(im_arr, dims=[\"y\", \"x\"])\n",
+    "\n",
+    "\n",
+    "def true_color_img(chip_id, data_dir=TRAIN_FEATURES):\n",
+    "    \"\"\"Given the path to the directory of Sentinel-2 chip feature images,\n",
+    "    plots the true color image\"\"\"\n",
+    "    chip_dir = data_dir / chip_id\n",
+    "    red = get_xarray(chip_dir / \"B04.tif\")\n",
```

Comment:
get_xarray can be replaced with rioxarray.open_rasterio

```suggestion
    "    red = rioxarray.open_rasterio(chip_dir / \"B04.tif\").squeeze()\n",
```
likewise for the other two.